crypto/dilithium/pqdsa.c

```diff
@@ -81,12 +85,37 @@ int PQDSA_KEY_set_raw_private_key(PQDSA_KEY *key, const uint8_t *in) {
   return 1;
 }
 
+DEFINE_LOCAL_DATA(PQDSA_METHOD, sig_ml_dsa_44_method) {
+  out->keygen = ml_dsa_44_keypair;
+  out->sign = ml_dsa_44_sign;
+  out->verify = ml_dsa_44_verify;
+}
+
 DEFINE_LOCAL_DATA(PQDSA_METHOD, sig_ml_dsa_65_method) {
   out->keygen = ml_dsa_65_keypair;
   out->sign = ml_dsa_65_sign;
   out->verify = ml_dsa_65_verify;
 }
 
+DEFINE_LOCAL_DATA(PQDSA_METHOD, sig_ml_dsa_87_method) {
+  out->keygen = ml_dsa_87_keypair;
+  out->sign = ml_dsa_87_sign;
+  out->verify = ml_dsa_87_verify;
+}
+
+DEFINE_LOCAL_DATA(PQDSA, sig_ml_dsa_44) {
+  out->nid = NID_MLDSA44;
+  out->oid = kOIDMLDSA44;
+  out->oid_len = sizeof(kOIDMLDSA44);
+  out->comment = "MLDSA44 ";
```

Comment:
I see that there's a trailing space here and that it's consistent with ML-DSA-65. Can you confirm that this is intentional?

Reply:
It was intentional, as the ML-KEM comments all have a space, I assumed it was needed (https://github.com/jakemas/aws-lc/blob/e4092fb224a2336b8e0d908f311924794d739042/crypto/fipsmodule/kem/kem.c#L137)

However, looking at the EC curves it appears it is not:
https://github.com/jakemas/aws-lc/blob/97a6706becf18605c83f526d4d08e90b74e7f3c6/crypto/fipsmodule/ec/ec.c#L118